BridgeCareApp/BridgeCare/Controllers/InvestmentStrategiesController.cs

```diff
@@ -22,8 +22,34 @@ public InvestmentStrategiesController(IInvestmentStrategies investmetStrategiesR
             db = context ?? throw new ArgumentNullException(nameof(context));
         }
 
-        // Get: api/InvestmentStrategies
+        ///<summary> Get: api/GetInvestmentStrategies
+        ///argument: NetworkModel
+        ///</summary>
         [ModelValidation("Given network data is not valid")]
-        public IQueryable<InvestmentStrategyModel> Get(NetworkModel network) => investmentStrategies.GetInvestmentStrategies(network, db);
+        [Route("api/GetInvestmentStrategies")]
+        [HttpGet]
+        public IQueryable<InvestmentStrategyModel> Get([FromBody]NetworkModel network) => investmentStrategies.GetInvestmentStrategies(network, db);
+
+        ///<summary> Post: api/SaveInvestmentStrategy
+        ///argument: InvestmentStrategyModel
+        ///return : 200 sucess
+        ///         400 for bad input argument
+        ///         500 internal server error (uncaught exception)
+        ///</summary>
+        [ModelValidation("Given investment strategy data is not valid")]
+        [Route("api/SaveInvestmentStrategy")]
+        [HttpPost]
+        public IHttpActionResult Post([FromBody]InvestmentStrategyModel data)
+        {
+            bool getResults = investmentStrategies.SetInvestmentStrategies(data, db);
```

Comment:
return investmentStrategies.SetInvestmentStrategies(data, db)) ? Ok() : NotFound();

creates an error : CS0173	Type of conditional expression cannot be determined because there is no implicit conversion between 'System.Web.Http.Results.OkResult' and 'System.Web.Http.Results.NotFoundResult'	BridgeCare	C:\Users\rbyrne\Documents\GitHub\iAM\BridgeCareApp\BridgeCare\Controllers\InvestmentStrategiesController.cs 

It is resolved by casting. Which I had not really looked into until now, or probably would have condensed further to:

public IHttpActionResult Post([FromBody]InvestmentStrategyModel data) 
            => investmentStrategies.SetInvestmentStrategies(data, db) ? (IHttpActionResult) Ok() : NotFound();